docs/app/pages/components/components-sections/tree-view/tree-view-custom-node/tree-view-custom-node.service.ts

```diff
@@ -1,7 +1,7 @@
 import { Injectable } from '@angular/core';
 import 'chance';
 
-@Injectable()
+@Injectable({ providedIn: 'root' })
 export class TreeViewService {
```

Comment:
These sections don't seem t be appearing:

![image](https://user-images.githubusercontent.com/20795331/75801298-cfab8200-5d72-11ea-9a37-1cb7e6ca9391.png)

Seeing this error:
![image](https://user-images.githubusercontent.com/20795331/75801334-df2acb00-5d72-11ea-96df-1fba26d62868.png)